monorepo/packages/pools/src/hooks/sorting.ts

```diff
@@ -0,0 +1,16 @@
+import { Token } from '@pangolindex/sdk';
+import { getTokenComparator } from '@pangolindex/shared';
+import { useAllTokenBalances } from '@pangolindex/state-hooks';
+import { useMemo } from 'react';
+
+export function useTokenComparator(inverted: boolean): (tokenA: Token, tokenB: Token) => number {
```

Comment:
We are already using `useTokenComparator` in swap also so we need to move this to somewhere in `state-hooks` something like in `state-hooks/src/state/wallet` so we dont need to manage 2 copies of same hook.